translations/binaries/slps_strings_city_names.json

```
@@ -0,0 +1,816 @@
+            "string": "Toukei[END]",
+            "translation": "Toukei[END]",
+            "original_bytes_length": 7,
+            "available_bytes_size": 7
+        },
+        {
+            "offset": 1731800,
+            "string": "トウケイ領[END]",
+            "original_bytes_length": 11,
+            "available_bytes_size": 15,
+            "translation": "토케이령[END]"
+        },
+        {
+            "offset": 1731816,
+            "string": "アクアヴェイルの北部にある[LINE]\nシデン家が治める領内の街。[END]",
+            "original_bytes_length": 54,
+            "available_bytes_size": 55,
+            "translation": "아쿠아베일의 북부에 위치한[LINE]\n시덴 가문이 다스리는 영내의 도시.[END]"
+        },
+        {
+            "offset": 1731872,
+            "string": "Shiden[END]",
+            "translation": "Shiden[END]",
+            "original_bytes_length": 7,
+            "available_bytes_size": 7
+        },
+        {
+            "offset": 1731880,
+            "string": "シデン領[END]",
+            "original_bytes_length": 9,
+            "available_bytes_size": 15,
+            "translation": "시덴령[END]"
+        },
+        {
+            "offset": 1731896,
+            "string": "ファンダリアの東部にある港町。[LINE]\n温泉があるため、他国からも湯治目的の[LINE]\n観光客が訪れる。[END]",
+            "original_bytes_length": 85,
+            "available_bytes_size": 87,
+            "translation": "팬더리아의 동부에 위치한 항구 도시.[LINE]\n온천이 있어, 다른 나라에서도 요양을 목적으로[LINE]\n관광객이 방문함.[END]"
+        },
+        {
+            "offset": 1731984,
+            "string": "Snowfreer[END]",
+            "translation": "Snowfreer[END]",
+            "original_bytes_length": 10,
+            "available_bytes_size": 15
+        },
+        {
+            "offset": 1732000,
+            "string": "港町スノーフリア[END]",
+            "original_bytes_length": 17,
+            "available_bytes_size": 23,
+            "translation": "항구 도시 스노우플리아[END]"
+        },
+        {
+            "offset": 1732024,
+            "string": "ファンダリアの南部にある街。[LINE]\n２年前に王家の軍が派遣されて街は戦場と[LINE]\n化したが、現在は復興を遂げている。[END]",
+            "original_bytes_length": 103,
+            "available_bytes_size": 103,
+            "translation": "팬더리아의 남부에 위치한 도시.[LINE]\n2년 전 왕가의 군대가 파견되어 도시는 전장이 되었지만,[LINE]\n현재는 복구를 이루고 있다.[END]"
+        },
+        {
+            "offset": 1732128,
+            "string": "Cyril[END]",
+            "translation": "Cyril[END]",
+            "original_bytes_length": 6,
+            "available_bytes_size": 7
+        },
+        {
+            "offset": 1732136,
+            "string": "サイリルの街[END]",
+            "original_bytes_length": 13,
+            "available_bytes_size": 15,
+            "translation": "사이릴의 도시[END]"
+        },
+        {
+            "offset": 1732152,
+            "string": "森の奥に整然とたたずむ神殿。[LINE]\n神殿としてだけでなく、研究施設としての[LINE]\n役割も担っている。[END]",
+            "original_bytes_length": 87,
+            "available_bytes_size": 87,
+            "translation": "숲 깊숙이 정돈되어 있는 신전.[LINE]\n신전으로만이 아니라 연구 시설로서의[LINE]\n역할도 맡고 있다.[END]"
```

```suggestion
            "translation": "숲 깊은 곳에 꼿꼿이 세워진 신전.[LINE]\n신전만이 아니라 연구 시설로서의[LINE]\n역할도 맡고 있다.[END]"
```

뭔가 좀 고고하고 질서 있고 잘 서있다는 느낌... 인데 더 적당한 단어가 생각이 안 나네요

---

```suggestion
            "translation": "숲 깊숙이 정연하게 자리 잡은 신전.[LINE]\n신전만이 아니라 연구 시설로서의[LINE]\n역할도 맡고 있다.[END]"
```

'정연하게' 어때요? 가지런하고 질서가 있다는 뜻인데, '질서정연하다'처럼도 쓰여요.

아니면 '정갈하게'?

```suggestion
            "translation": "숲 깊숙이 정갈하게 자리 잡은 신전.[LINE]\n신전만이 아니라 연구 시설로서의[LINE]\n역할도 맡고 있다.[END]"
```

---

원문 한자가 정연이니만큼 정연하게가 좋을 것 같네요!